recipes/livekraken/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+mkdir -p "$PREFIX/libexec" "$PREFIX/bin"
```

Comment:
Can you use `$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM` as documented [here](https://bioconda.github.io/guidelines.html#java)? Otherwise anything in `libexec` could get clobbered, such as the generically-named  [`upgrade_db.sh`](https://gitlab.com/SimonHTausch/LiveKraken/blob/master/scripts/upgrade_db.sh).

I see that you mirrored the existing `kraken` recipe, but I think that one could be fixed as well (eventually).

Reply:
Thanks for the advice. I adapted the paths.